1-js/11-async/08-async-await/article.md

```diff
@@ -286,22 +284,22 @@ let results = await Promise.all([
 ]);
 ```
 
-In case of an error, it propagates as usual: from the failed promise to `Promise.all`, and then becomes an exception that we can catch using `try..catch` around the call.
+In the case of an error, it propagates as usual, from the failed promise to `Promise.all`, and then becomes an exception that we can catch using `try..catch` around the call.
 
 ````
 
 ## Summary
 
 The `async` keyword before a function has two effects:
 
-1. Makes it always return a promise.
-2. Allows to use `await` in it.
+1. Makes it always return a promise
```

Comment:
Wording change OK, is an improvement.

But removing periods from bullet-points in summary sections is not, from what I've noticed, standard for this document.

Reply:
Good point